### DISCUSSION TITLE
How are you using Laravel Ban?

### DISCUSSION THREAD

**Discussion by antonkomarev** [cybercog/laravel-ban#70] [Show and tell]
Please leave a comment explaining how you are using this package.
Feel free to share links on your public projects.
Any feedback will be helpful when developing the new version!

All these questions are optional, but might help to find bottlenecks of the package:
- How many users are you supporting?
- What type of features have you had to implement to extend the base functionality of this package?

*If you have any issues, please open up a new ticket. Please keep this thread for feedback only. Thanks!*

**Comment by Im-Fran** [comment]
I'm developing a website to sell and share my software (mostly minecraft plugins), in this website I'm trying to create a community, so to keep it clean I wanted to add bans to my website, this package helped me a lot!
This package is awesome if you ask me.

**Comment by hamzaelmaghari** [comment]
Just a classic function inside resource controller, added button in a table of admin -> requesting the $id of the user then BAN!
By default in the view I use if-else condition if user banned -> disaplay UNBAN btn. else disaplay BAN btn.
Thanks for this amazing package. looks awesome.
